src/lancer.scss

```diff
@@ -2432,8 +2433,9 @@ a.activation-chip > i.mdi {
 .track-message.lancer-action-grid {
   flex-wrap: wrap;
 }
-.track-message .lancer-action-badge {
-  margin: 4px;
+.lancer-action-badge {
+  background-color: var(--light-gray-color);
+  margin: 3px;
```

Comment:
The reason I removed `.track-message` here is to reduce specificity so that `button.lancer-#{$theme-type}` takes precedence for `background-color`. (So gray vs the action's color).

`.lancer-action-badge` is not used elsewhere, so removing `.track-message` here seems safe.